index.html

```diff
@@ -9,21 +9,31 @@
 ---
 
 <!-- Hero Banner -->
-<section role="region" aria-label="hero-banner" class="ds-l-container ds-u-padding--1 ds-u-sm-padding--2 ds-u-md-padding--3 ds-u-lg-padding--4 ds-u-xl-padding--5">
-  <div class="ds-l-row ds-u-align-items--center">
-    <div class="ds-l-sm-col--12 ds-l-md-col--12 ds-l-lg-col--6 ds-u-text-align--center">
-      <div class="ds-u-font-size--h1 ds-u-leading--base ds-u-font-weight--bold ds-u-padding-y--3 ds-u-text-transform--uppercase">
+<section role="region" aria-label="hero-banner" class="ds-l-container ds-u-padding--1 ds-u-sm-padding--2 ds-u-md-padding--3 ds-u-lg-padding--4 ds-u-xl-padding--5 ds-l-container">
```

Comment:
just checking in on this one, I know you and Ali did some coordination -- when I run the website locally, this part looks different from the wireframe, are we still going with the wireframe or did Ali give different feedback?

Running locally:
![image](https://github.com/CMSgov/bcda-static-site/assets/126501259/9fd557b2-ff1f-4b42-8132-6bb2754aedc6)

Wireframe:
<img width="890" alt="image" src="https://github.com/CMSgov/bcda-static-site/assets/126501259/81cd7c1e-be41-4a62-866c-072c5820a71c">

Reply:
Hi @laurenkrugen-navapbc Ali suggested this change!

Reply:
Thanks for the context! 😄